README.md

```diff
@@ -2,14 +2,21 @@
 
 This is an audio module built on top of FastAI to allow you to quickly and easily build machine learning models for a wide variety of audio applications. We are an unofficial library and have no official connection to fastai except that we are huge fans and want to help make their tools more widely available and applicable to audio.
 
+# Dependencies
+Currently uses Pytorch 1.30, FastAI 1.0.59, Librosa 0.7.1
+```
+conda install -c pytorch -c fastai fastai
+conda install -c pytorch torchaudio
+conda install -c conda-forge librosa
+```
+
+
 # Installation
 To install, run the following commands in a shell. 
 
 ```
 git clone https://github.com/mogwai/fastai_audio
 cd fastai_audio
-sudo chmod +x install.sh
```

Comment:
Why was the install script replaced with conda installs?

Reply:
The reason I changed it was that pytorch and fastai are installed via conda and the conda docs recommend avoiding installing packages via pip if conda was used.  See https://www.anaconda.com/using-pip-in-a-conda-environment/

Also, based on my install  fastai_audio only needed  a conda install of librosa and then it ran fine through all the tutorial workbooks.  Worth a second set of eyes tho.

Reply:
For now I think just keep the install script its just an easy way for people to get up and running and documents what the requirements are. I agree @kevinbird15  that a envrionment.yml would be better but we're not planning on supporting this version of the library when v2 is complete. 

Add the install.sh back into the readme and change the install script to install torchaudio==0.3.1 on the last line and we can merge this PR.

Great job @filipmu thank you